test.php

```diff
@@ -10,7 +10,7 @@
  * @link     5
  */
```

Comment:
:no_entry_sign: **Error**: There must be no blank lines after the function comment (*Squiz.Commenting.FunctionComment.SpacingAfter*).